src/building/suggested.md

```diff
@@ -339,3 +339,8 @@ pkgs.mkShell {
   RUST_BOOTSTRAP_CONFIG = config;
 }
 ```
+
+## Shell Completions
+
+If you use Bash, Fish or PowerShell, you can find automatically-generated shell completion scripts for `x.py` in [`src/etc/completions`](https://github.com/rust-lang/rust/tree/master/src/etc/completions).
```

Comment:
Can we also document how to use the scripts? is it as simple as `source src/etc/completions/bash.sh`?

Reply:
I'll add some quick suggestions